packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -2,13 +2,18 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
-
 #include "App.h"
 #include "ReactPackageProvider.h"
 #include "ReactPropertyBag.h"
 #include "winrt/SampleLibraryCS.h"
 #include "winrt/SampleLibraryCpp.h"
 
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
```

Comment:
use xaml alias for winui3 friendliness :)

Reply:
Changed to use the xaml alias.

---
In reply to: [432992769](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432992769) [](ancestors = 432992769)